dev/airflow-pr

```diff
@@ -443,17 +445,6 @@ def resolve_jira_issue(comment=None, jira_id=None, merge_branches=None):
         raise ValueError(
             "ASF JIRA could not find issue {}\n{}".format(jira_id, e))
 
-    if comment is None:
```

Comment:
Note: these comment lines are more appropriate after the "Proceed with Airflow-211?" question, so this just moves them down.